src/modules/home/blogs/blogCard.tsx

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import { BlogCardProps } from "./helper";
+
+export const Blog = ({
+  name,
+  date,
+  writer_image,
```

Comment:
please refactor it to use `camelCase`, we should stick to one naming convention, and since camelCase is mostly use with react, let's use that

Reply:
oh yes. My bad